teamworkApp/app/views/pages/home.html.erb

```diff
@@ -23,6 +23,11 @@
             <div align="left"><%= render 'pages/importdatashow.html.erb'%></div>
         </div>
       </div>
+      <div class="row" style="padding: 0px">
+        <center>
+          <a href="python/deleteData" class="btn btn-warning" role="button">Remove All Data (Data will be lost)</a>
```

Comment:
Nit: Could you put (Data will be lost) on a separate line so the button won't be so long? Could it also be centered?

Reply:
Yeah I'll figure out the new line, also the picture is pretty bad, but the button is centered